mocha-config.json

```diff
@@ -1,6 +1,9 @@
 {
-    "reporterEnabled": "mocha-junit-reporter, spec",
+    "reporterEnabled": "mocha-junit-reporter, spec, mocha-sonarqube-reporter",
```

Comment:
Sonar is still capable of pulling in test results, but it requires a really quirky format which requires special reporter. This one will work, see [our gulp plugin here](https://sonarcloud.io/component_measures?id=swellaby%3Agulp-vsts-bump&metric=tests) but it does require the inclusion of the .js files in the scan

Reply:
Understandable, all good.